### PR TITLE
Added QR codes on receive page

### PR DESCRIPTION
**Pull request by Jaqueeee**
Added QR codes as instructed in https://github.com/mbg033/monero-core/issues/8

Clipboard icon overlapping the address reported in https://github.com/mbg033/monero-core/issues/34#issuecomment-238605971

### CODE REVIEW

File: components/qqr.js [hunk 1]
```diff
@@ -0,0 +1,1198 @@
+// [qr.js](http://neocotic.com/qr.js)  
+// (c) 2014 Alasdair Mercer  
+// Licensed under the GPL Version 3 license.
```
Comment (mbg033) [inline components/qqr.js:3]: Sorry, but (AFAIK) monero license is not compatible with GPL

**Comment by mbg033** [comment]
Cherry-picked   540481d for now

File: components/qqr.js (hunk 1 above)
Comment (dEBRUYNE-1) [reply components/qqr.js:3]: Correct, would need to be a non gpl licensed version (e.g. bsd or mit licensed).

Comment (Jaqueeee) [reply components/qqr.js:3]: Talked to Othe about using this library: https://github.com/fukuchi/libqrencode
Although we have to add it as a dependency and not include it in the repo.

**Pull request closed without merging**